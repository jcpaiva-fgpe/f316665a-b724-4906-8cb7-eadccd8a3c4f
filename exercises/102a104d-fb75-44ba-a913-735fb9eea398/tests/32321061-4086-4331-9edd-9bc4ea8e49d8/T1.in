3(x+2)-4{x+[2y-(27-z)+8w]}-1
