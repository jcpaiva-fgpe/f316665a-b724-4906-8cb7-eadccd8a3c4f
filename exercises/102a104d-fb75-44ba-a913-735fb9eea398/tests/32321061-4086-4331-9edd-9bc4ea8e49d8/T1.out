() 1 5
() 15 20
[] 11 24
{} 8 25
